components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -207,6 +208,10 @@ def generate_credentials_file(credentials_file_path: pathlib.Path):
             'user': 'clp-user',
             'password': secrets.token_urlsafe(8)
         },
+        REDIS_COMPONENT_NAME: {
+            # redis can perform authentication without a username
```

Comment:
How about moving this comment to the Redis pydantic object (feels like that's where people would look first).

Reply:
Sure thing